### PR TITLE
Super simple client error logging

### PR DESCRIPTION
**Pull request by ajacksified**
:eyeglasses: @schwers 

This adds the ability for the client to send error logs to the server, which will be `console.log`. You can then grep the server logs for output to see what's going on on the client. Includes stack traces if available.

### CODE REVIEW

File: src/lib/errorLog.es6.js [hunk 1]
```diff
@@ -49,7 +49,6 @@ function formatLog(details) {
 
 function errorLog(details, errorEndpoints, config={}) {
   let formattedLog = formatLog(details);
-  console.log(formattedLog);
```
Comment (schwers) [inline src/lib/errorLog.es6.js]: We should probably keep this log

File: assets/js/client.es6.js [hunk 2]
```diff
@@ -56,6 +62,8 @@ var $body = document.body || document.getElementsByTagName('body')[0];
 var $head = document.head || document.getElementsByTagName('head')[0];
 
 var config = defaultConfig();
+// the client should post errors to the /error endpoint.
+config.postErrorURL = POST_ERROR_URL;
```
Comment (schwers) [inline assets/js/client.es6.js]: Should this just be in `config`? Then `routes.es6.js` could use it too

**Comment by schwers** [comment]
:nail_care:

**Comment by schwers** [comment]
:fish:

**Comment by ajacksified** [comment]
:fish:

**Comment by ajacksified** [comment]
:eyeglasses: @ajacksified

**Pull request merged**